src/pybind/mgr/dashboard/frontend/cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "http://localhost:4200/",
+  "baseUrl": "https://localhost:4200/",
```

Comment:
If you check the `CYPRESS_BASE_URL` env. variable set from `.../dashboard/run-frontend-e2e-tests.sh`, I guess this is overridden by that env. var.